src/styles/theme/defaultTheme.ts

```diff
@@ -398,19 +398,6 @@ export const defaultTheme = createTheme({
         },
       },
     },
-    MuiBreadcrumbs: {
-      styleOverrides: {
-        li: {
-          fontSize: '18px',
-        },
-        separator: {
-          color: COLORS.brandDark,
```

Comment:
I don't see this color being applied to the breadcrumb separator component. Does it automatically have the same color?